static/js/controllers/configuration-forms.js

```diff
@@ -89,10 +89,10 @@ angular.module('inboxControllers').controller('ConfigurationFormsCtrl',
           }
 
           var update = function(doc) {
-            doc.context = context;
             doc.title = title;
             doc.type = 'form';
             doc.internalId = formId;
+            _.extend(doc, meta);
```

Comment:
Ok. I'll move the `doc.type` line down so it's always a "form".

I notice now that although the upload script does let you overwrite the internal ID it's not recommended due to #3342 . This is probably a good time to start enforcing it so I'll move that line down too and add a note to the changelog.

Overwriting the title is actually really useful for internationalisation so I'll leave that one in.